pytorch_to_returnn/torch/nn/modules/rnn.py

```diff
@@ -29,16 +29,25 @@ class PackedSequence(PackedSequence_):
   always be satisfied.
   """
 
-  def get_batched_tensor(self) -> Tensor:
+  def get_batched_tensor(self, batch_first=None) -> Tensor:
```

Comment:
Maybe we should rename this function to `get_padded_tensor` to make it more clear.

Reply:
You should also specify the type of `batch_first` (`Optional[bool]`).

Reply:
Btw, do you want to allow `batch_first` to be `None` here? Where do we call this `get_batched_tensor` (`get_padded_tensor`)? The resulting Torch tensor, maybe we always want to have it explicit whether it is batch major or not?

Reply:
The renaming makes it also more consistent with the PyTorch names. We call this function in `pad_packed_sequence` and in the `LSTM` module. I'm fine with making it always explicit, this is probably clearer.